code/__HELPERS/mobs.dm

```diff
@@ -107,6 +113,13 @@
 		if(i != attempts_to_find_unique_name && !findname(.))
 			break
 
+/proc/random_unique_fox_name(gender, attempts_to_find_unique_name=10)
+	for(var/i=1, i<=attempts_to_find_unique_name, i++)
+		. = capitalize(lizard_name(gender))

```

Comment:
the proc is for a `random unique fox name` - but returned lizard_name?

Also, fox?